governance-proposal/TSC-Project-Lifecycle.md

```diff
@@ -0,0 +1,78 @@
+
+The TSC shall have the authority to grandfather proposed Projects with a significant history and record to meet the project state descriptions below into a higher order state during the first 12 months after the first TSC meeting.
+
+| Project state |	Description |
+| ------------- | ----------- |
+| Proposal | Project does not formally exist yet, may not have real resources (yet), but is being worked on by the community to submit a formal proposal to the TSC. |
+| Incubation | Project has been approved by the TSC, has resources, but is recognized to be nascent. |
+| Mature | Project is fully functioning and stable, has achieved successful releases, has a Project Lead, but is not a required component of the platform.|
+| Core | Project is a required component of the Node.js platform.|
+| Archived | Project has been recognized as no longer being actively used or developed. This could be for a variety of reasons, e.g. project successfully accomplished its goals but is no longer used, project failed, etc., and has been archived as it's no longer a going concern.|
+ 
+### Project state transitions
+
+| From State | To State | Review |
+| ---------- | -------- | ------ |
+| `null` | Proposal | n/a |
+| Proposal | Incubation | Creation Review |
+| Incubation | Mature | Graduation Review |
+| Mature | Core | Core Review |
+| Proposal, Incubation, Mature, Core | Archived | Archive Review |
+
+## Reviews
+
+### Creation Review
+* Proposal posted for two weeks, evaluated on metrics of:
+ * Name is okay (e.g. no use of a trademark)
+ * Project contact name and email
+ * Description is complete
+ * Scope and project plan is well defined
+ * Resources are committed
+ * Initial Committers named
+ * Contributors have been identified
+ * Meets Foundation’s policies (e.g. IP Policy)
+ * Proposal has been socialized with potentially interested or affected existing Projects
+ * Proposal email has been sent to the TSC mailing list
+* Review by TSC: Confirm that the proposal is complete and the above listed requirements have been sufficiently met.
+
+### Graduation Review
+* Graduation proposal posted for two weeks:
+ * The Project demonstrates stable output (code base, documents, tests)
+ * Active community working on the Project
+ * History of successful, consistent releases in accordance with the release process
+* TSC review
+ * Working and stable code base exists
+ * Active community exists
+ * Project has demonstrated a history of releases following the release process and cadence
+ * Confirmed acceptance and successful integration of contributions/code to partner/upstream projects. 
+ * Testing/integration environment defined and mature, tests and integration run successfully
+ * Detailed documentation available documenting the code
+* Core Review
```

Comment:
It looks to me like the `Core Review` should be a level 3 heading rather than a bullet under `Graduation Review`:

### Graduation Review
* Graduation proposal posted for two weeks:
* ...

### Core Review
* Core-state proposal posted for two weeks
* ...

### Termination Review
* Termination proposal posted for two weeks
* ...